pkg/limayaml/limayaml.go

```diff
@@ -182,6 +182,11 @@ type Firmware struct {
 	// LegacyBIOS is ignored for aarch64.
 	LegacyBIOS *bool `yaml:"legacyBIOS,omitempty" json:"legacyBIOS,omitempty" jsonschema:"nullable"`
 
+	// CompatUEFIViaBIOS forces QEMU to load concatenated firmware with -bios option
+	// CompatUEFIViaBIOS is ignored for non x86_64
+	// This should be deprecated, if the issue in QEMU is fixed https://gitlab.com/qemu-project/qemu/-/issues/513
```

Comment:
Since this is already deprecated since its birth, may this should rather be an env var?
Similar to `LIMA_SSH_PORT_FORWARDER` (originally planned to be deprecated in v1.0, now postponed to v1.1)
https://lima-vm.io/docs/config/environment-variables/

Reply:
I agree. It might never be removed, because fixes in Windows QEMU version are not of the highest priority. If the ergonomics of the environment var would be insufficient it could be changed later into full-fledged option in Lima yaml. I'm moving this into draft to rework.